endpoints/tags.py

```diff
@@ -0,0 +1,92 @@
+import json
+from subprocess import run, Popen, PIPE, STDOUT
+from utils.attacker import Attacker
+from utils.util import print_header
+
+
+class Tags:
+    """
+    Contains all the functions to interact with tags and catalog endpoints.
+    """
+
+    def __init__(self):
+        pass
+    
+    @staticmethod
+    def fetch_repo_tokens(quay_host, user, repo):
+        """
+        Fetches v2 token for a given repo of the user.
+        
+        :param quay_host: quay host name
+        :param user: username
+        :param repo: repository of a specified user
+        :return: returns the token
+        """
+        cmd = ["curl", "-k", "-L", "--user", f"{user}:password", f"https://{quay_host}/v2/auth?service={quay_host}&scope=repository:{user}/{repo}:pull,push"]
```

Comment:
Why do you use `curl` instead of doing it in Python via urllib or requests?

Reply:
Thanks for bringing this up. Made changes to use python `requests` library.